docs/using-concourse/teams.any

```diff
@@ -266,11 +268,24 @@ as).
           --generic-oauth-client-secret $CLIENT_SECRET \\
           --generic-oauth-auth-url https://oauth.example.com/o/oauth2/auth \\
           --generic-oauth-token-url https://oauth.example.com/o/oauth2/token \\
-          --generic-oauth-auth-url-param=scope:read
+          --generic-oauth-auth-url-param scope:read
```

Comment:
It's related in a way. During the OAuth process, the client can request a desired set of scopes. Some OAuth servers will simply fail the request if you don't request any scopes. When the user token is returned, that token includes the scopes that the user actually has. This is what the `-scope` flag is checking for verification.